addons/website_event/static/src/js/website_event.js

```diff
@@ -23,7 +23,8 @@ var EventRegistrationForm = Widget.extend({
                 .off('click')
                 .click(function (ev) {
                     self.on_click(ev);
-                });
+                })
+                .closest('[type="submit"]').prop('disabled', false);
```

Comment:
Isn't the button already the selected element (and the one we click on)?
```suggestion
                })
                .prop('disabled', false);
```

Reply:
didn't notice, code updated on the matter :)